### PR TITLE
Add single page ui

### PR DESCRIPTION
**Pull request by faberald**
Need Improve Search

### CODE REVIEW

File: service/static/index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                <label class="control-label col-sm-2" for="restock_level">Restock Level:</label>
+                <div class="col-sm-10">
+                    <input type="text" class="form-control" id="restock_level" placeholder="Enter the restock level for the Inventory">
+                  </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="condition">Condition:</label>
+                <div class="col-sm-10">
+                  <select class="form-control" id="condition">
+                        <option value="" selected>Select the condition for the inventory</option>
```
Comment (eatingzhang0611) [inline service/static/index.html]: Can add disable in option, so that this option will never be selected again.

File: service/static/index.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                        <option value="open_box">Open Box</option>
+                        <option value="used">Used</option>
+                  </select>
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="available">Availability:</label>
+                <div class="col-sm-10">
+                  <select class="form-control" id="available">
+                        <option value="" selected>Select the availability for the inventory</option>
```
Comment (eatingzhang0611) [inline service/static/index.html]: Here as well
`<option value="" selected disabled> `

Comment (joanne3634) [reply service/static/index.html]: we need this option for user to set back to empty value

Comment (eatingzhang0611) [reply service/static/index.html]: Got it! Thanks!

**Comment by faberald** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/42ed7663-cc42-4378-b41b-ba0fb0a42b26?env_id=ibm%3Ayp%3Aus-south "Inventory-Toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/5ef0d78c-44b1-4115-aa5d-8283a9ae61db "inventory") deployed [inventory](https://github.com/Inventory-Squad/inventory/releases/tag/deploy-ibm-yp-us-south-hl3635-nyu.edu-dev-20191117-074034 "deploy-ibm-yp-us-south-hl3635-nyu.edu-dev-20191117-074034") to [dev](https://console.bluemix.net/apps/42751834-e78d-4af5-af38-81732250aad5?env_id=ibm:yp:us-south "ibm:yp:us-south:hl3635@nyu.edu:dev"), including commit 44df22b961f0fbd49665660960919ff6b219b8a6

**Comment by faberald** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/42ed7663-cc42-4378-b41b-ba0fb0a42b26?env_id=ibm%3Ayp%3Aus-south "Inventory-Toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/5ef0d78c-44b1-4115-aa5d-8283a9ae61db "inventory") deployed [inventory](https://github.com/Inventory-Squad/inventory/releases/tag/deploy-ibm-yp-us-south-hl3635-nyu.edu-dev-20191124-081957 "deploy-ibm-yp-us-south-hl3635-nyu.edu-dev-20191124-081957") to [dev](https://console.bluemix.net/apps/42751834-e78d-4af5-af38-81732250aad5?env_id=ibm:yp:us-south "ibm:yp:us-south:hl3635@nyu.edu:dev"), including commit 44df22b961f0fbd49665660960919ff6b219b8a6

**Comment by faberald** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/42ed7663-cc42-4378-b41b-ba0fb0a42b26?env_id=ibm%3Ayp%3Aus-south "Inventory-Toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/5ef0d78c-44b1-4115-aa5d-8283a9ae61db "inventory") deployed [inventory](https://github.com/Inventory-Squad/inventory/releases/tag/deploy-ibm-yp-us-south-hl3635-nyu.edu-prod-20191124-174147 "deploy-ibm-yp-us-south-hl3635-nyu.edu-prod-20191124-174147") to [prod](https://console.bluemix.net/apps/39722351-ab93-4688-93da-a6b43e04647f?env_id=ibm:yp:us-south "ibm:yp:us-south:hl3635@nyu.edu:prod"), including commit 44df22b961f0fbd49665660960919ff6b219b8a6

**Pull request merged**